ova/ovf/centos.ovf

```diff
@@ -138,6 +138,7 @@ file.
       <vmw:Config ovf:required="false" vmw:key="firmware" vmw:value="bios"/>
       <vmw:Config ovf:required="false" vmw:key="virtualICH7MPresent" vmw:value="false"/>
       <vmw:ExtraConfig ovf:required="false" vmw:key="nvram" vmw:value="ovf:/file/file2"/>
+      <vmw:ExtraConfig ovf:required="false" vmw:key="disk.EnableUUID" vmw:value="1"/>
```

Comment:
Didn't notice the value when I first looked, but this is a boolean property - should be `"true"` instead, right?